site/content/contribute/desktop/debugging.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Debugging"
+date: 2019-01-22T00:00:00-05:00
+weight: 3
+subsection: "Debugging"
+---
+
+# Developer Tools
+
+The electron app itself can be inspected using the developer tools, available from the View menu:
+
+![image](/contribute/desktop/view-toggle-developer-tools.png)
+
+However, this view does not expose the contents of the webviews for inspection. To open the developer tools for a webview, first access the console for the electron shell as above, and then run
+
+    document.getElementById("mattermostView0").openDevTools();
```

Comment:
Not important, but just for consistency with other codeblocks?

```suggestion
```js
document.getElementById("mattermostView0").openDevTools();
```.
```